contracts/pcv/MigratorRouter.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity =0.8.13;
+
+import {IMigratorRouter} from "./IMigratorRouter.sol";
+import {IPegStabilityModule} from "../peg/IPegStabilityModule.sol";
+import {IVolt} from "../volt/IVolt.sol";
+import {IVoltMigrator} from "../volt/IVoltMigrator.sol";
+
+/// @title Migrator Router
+/// @notice This contract is a router that wraps around the token migrator from
+/// the old volt ERC20 token to the new ERC20 token, to allow users to redeem for
+/// stables using the old volt version once the new version is live
+contract MigratorRouter is IMigratorRouter {
+    /// @notice the old VOLT token
+    IVolt public constant oldVolt =
+        IVolt(0x559eBC30b0E58a45Cc9fF573f77EF1e5eb1b3E18);
+
+    /// @notice the VOLT migrator contract
+    IVoltMigrator public constant voltMigrator =
+        IVoltMigrator(0x559eBC30b0E58a45Cc9fF573f77EF1e5eb1b3E18); // fill with correct address once deployed
+
+    /// @notice the VOLT-DAI PSM to swap between the two assets
+    IPegStabilityModule public constant daiPSM =
+        IPegStabilityModule(0x42ea9cC945fca2dfFd0beBb7e9B3022f134d9Bdd);
+
+    /// @notice the VOLT-USDC PSM to swap between the two assets
+    IPegStabilityModule public constant usdcPSM =
+        IPegStabilityModule(0x0b9A7EA2FCA868C93640Dd77cF44df335095F501);
```

Comment:
On construction, this contract should allow the migrator to spend all of its old volt